provision-osx/tasks/install_nodejs.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+-
+  name: Copy nvm install script
+  get_url: 
+    url: "{{nvm_install_url}}"
+    dest: "{{remote_tmp_path}}/{{nvm_install_file_name}}"
+  environment:
```

Comment:
I think there's a benefit to having it externally (the user can just specify a different release version with the URL and go) over the benefit of bundling it, especially since these scripts (nvm, rvm, brew) curl out anyway to other resources, so bundling them doesn't really help.

A list of the hosts that are called out to will need to be compiled and documented though.